test/cucumber_test.go

```diff
@@ -110,20 +111,3 @@ func InitializeScenario(ctx *godog.ScenarioContext) {
 		return ctx, nil
 	})
 }
-
-func TestFeatures(t *testing.T) {
```

Comment:
removed this to stop tests running twice once with `make test` and again with `make integration-test`